plugins/GSdx/GSRendererOGL.cpp

```diff
@@ -606,6 +597,32 @@ void GSRendererOGL::SendDraw(bool require_barrier)
 		ASSERT(GLLoader::found_GL_ARB_texture_barrier);
 		glTextureBarrier();
 		dev->DrawIndexedPrimitive();
+	} else if (m_vt.m_primclass == GS_SPRITE_CLASS) {
+		size_t nb_vertex = (GLLoader::found_geometry_shader) ? 2 : 6;
+
+		GL_PUSH("Split the draw");
+
+#if defined(ENABLE_OGL_DEBUG)
+		// Some code to check how draw code is split.
+		map<size_t, size_t> frequency;
+		for (auto it = m_drawlist.begin(); it != m_drawlist.end(); ++it)
```

Comment:
Now that we droped VS2012, we could also use the new range C++11 syntax. It is up to you. Same for the loop below. By the way, maybe be better to use a reference `auto& it`